package.json

```diff
@@ -107,6 +107,9 @@
     "webpack-dev-server": "^3.10.3"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
```

Comment:
There may be a larger plan here of which I'm unaware, but it looks like you're adding these several libraries as a means of getting access to a single fontawesome lock icon, which seems like a bit of overkill. What we did for the `slate-editor` work (and I think @knowuh did for some of the CLUE teacher dashboard work) was to download the SVG (e.g. this [fontawesome lock icon](https://www.iconfinder.com/icons/1608769/lock_icon)) and then convert it to a React component using the [SVGR](https://react-svgr.com/playground/) converter which results in a simple React component like this:
```
import * as React from "react"

function SvgComponent(props) {
  return (
    <svg height={1792} viewBox="0 0 1792 1792" width={1792} {...props}>
      <path d="M640 768h512V576q0-106-75-181t-181-75-181 75-75 181v192zm832 96v576q0 40-28 68t-68 28H416q-40 0-68-28t-28-68V864q0-40 28-68t68-28h32V576q0-184 132-316t316-132 316 132 132 316v192h32q40 0 68 28t28 68z" />
    </svg>
  )
}

export default SvgComponent
```
You can then add types for use with TypeScript and props (e.g. `fill`) and you end up with a component for each icon you need without the overhead of any additional dependencies. Just another option to consider.

Reply:
Right, I've been using SVGR each time Micheal is providing custom icons. It's nice then, but quite often I need to manually adjust SVG, e.g. viewbox or some little details. Not much work, but still. I'd prefer if we could pick one icon library for life. 😉 

Here I was assuming that these libraries will be only temporary - we don't have any design yet. But at some point we might end up with custom icons too. react-fontawesome looked good, as it was super simple to use and they're promising in readme that the way I imported lock icon, would import only this one SVG. So, I hoped it won't increase build size.

But since you mentioned that, I did a little test. Adding this single lock icon increases bundle size of all the interactives from 400KB to 1.07MB...  So, given that we'll have many of them on one LARA page, I think it's definitely worth refactoring for SVGR. Thanks.